src/java/org/jivesoftware/openfire/admin/JDBCAdminProvider.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * $Revision$
+ * $Date$
+ *
+ * Copyright (C) 2005-2008 Jive Software. All rights reserved.
```

Comment:
Yeah, so just add your own copyright statement. But don't put "All Rights Reserved" - just "Copyright 2012-2016 Robert Marcano" (in this example, I'm assuming you wrote it originally around 2012).